### PR TITLE
Update default control plane instance type to r6i.large

### PR DESCRIPTION
**Pull request by JosephSalisbury**
### What this PR does / why we need it

Towards https://github.com/giantswarm/giantswarm/issues/30671
Towards https://github.com/giantswarm/giantswarm/issues/30682

This PR updates the default instance type for control plane and worker nodes to be more cost effective. `r6i.large` is around 1/2 the price of `r6i.xlarge`, and can adequately provide enough resources for a default cluster.

### Checklist

- [x] Updated CHANGELOG.md.

### Trigger E2E tests

/run cluster-test-suites

### CODE REVIEW

File: helm/cluster-aws/ci/ci-values.yaml [hunk 1]
```diff
@@ -9,7 +9,6 @@ global:
       maxSize: 2
       minSize: 2
       instanceTypeOverrides:
-        - r6i.xlarge
```
Comment (JosephSalisbury) [inline helm/cluster-aws/ci/ci-values.yaml:12]: i don't know what this does but i got rid of it lol

File: helm/cluster-aws/values.schema.json [hunk 2]
```diff
@@ -950,7 +950,7 @@
                         "instanceType": {
                             "type": "string",
                             "title": "EC2 instance type",
-                            "default": "r6i.xlarge"
+                            "default": "m7i.large"
```
[the remark this answers is not in the document]
Comment (JosephSalisbury) [reply helm/cluster-aws/values.schema.json:953]: - sticking with `i` for IOPs for etcd (hence no `m5.large`)
- we could probably change to `m6i.large`, but the cost difference is very small, so i'd rather use the latest generation

**Comment by marians** [comment]
Is the intent here to change the default type for all nodes, CP and worker?

**Comment by JosephSalisbury** [comment]
@marians good shout, updating for workers too, let's just do this all at once

**Review by AndiDog — APPROVED** [review]

**Comment by JosephSalisbury** [comment]
okay, we have reserved instance considerations, so i'm actually going to change the default to r6i.large (from r6i.xlarge). it's about half the price, and we'll still be covered by our existing RIs (cc @paurosello)

we'll work out how to get to a cheaper instance type later

**Comment by JosephSalisbury** [comment]
```
2024-05-01 14:14:53 
$ KUBECONFIG=~/Desktop/kubeconfig kubectl resource-capacity --node-labels=kubernetes.io/role=master -p --sort cpu.request -u    
NODE                                         NAMESPACE     POD                                                                  CPU REQUESTS   CPU LIMITS     CPU UTIL     MEMORY REQUESTS   MEMORY LIMITS   MEMORY UTIL
*                                            *             *                                                                    3945m (93%)    3750m (89%)    657m (15%)   4010Mi (9%)       6300Mi (15%)    9962Mi (23%)
                                                                                                                                                                                                             
ip-10-0-235-76.eu-west-2.compute.internal    *             *                                                                    1380m (98%)    550m (39%)     248m (17%)   1270Mi (9%)       1950Mi (14%)    3653Mi (26%)
ip-10-0-235-76.eu-west-2.compute.internal    kube-system   kube-apiserver-ip-10-0-235-76.eu-west-2.compute.internal             250m (17%)     0m (0%)        98m (7%)     0Mi (0%)          0Mi (0%)        1332Mi (9%)
ip-10-0-235-76.eu-west-2.compute.internal    kube-system   aws-cloud-controller-manager-t4psf                                   200m (14%)     400m (28%)     3m (0%)      300Mi (2%)        600Mi (4%)      24Mi (0%)
ip-10-0-235-76.eu-west-2.compute.internal    kube-system   kube-controller-manager-ip-10-0-235-76.eu-west-2.compute.internal    200m (14%)     0m (0%)        27m (1%)     0Mi (0%)          0Mi (0%)        108Mi (0%)
ip-10-0-235-76.eu-west-2.compute.internal    kube-system   cluster-autoscaler-f64fc6bb9-k4sth                                   200m (14%)     0m (0%)        16m (1%)     400Mi (2%)        400Mi (2%)      67Mi (0%)
ip-10-0-235-76.eu-west-2.compute.internal    kube-system   etcd-ip-10-0-235-76.eu-west-2.compute.internal                       100m (7%)      0m (0%)        40m (2%)     100Mi (0%)        0Mi (0%)        312Mi (2%)
ip-10-0-235-76.eu-west-2.compute.internal    kube-system   k8s-dns-node-cache-xkmwq                                             100m (7%)      0m (0%)        1m (0%)      100Mi (0%)        0Mi (0%)        24Mi (0%)
ip-10-0-235-76.eu-west-2.compute.internal    kube-system   kube-scheduler-ip-10-0-235-76.eu-west-2.compute.internal             100m (7%)      0m (0%)        4m (0%)      0Mi (0%)          0Mi (0%)        36Mi (0%)
ip-10-0-235-76.eu-west-2.compute.internal    kube-system   cilium-mvzk5                                                         100m (7%)      0m (0%)        6m (0%)      10Mi (0%)         0Mi (0%)        205Mi (1%)
ip-10-0-235-76.eu-west-2.compute.internal    kube-system   node-exporter-node-exporter-l8vvn                                    50m (3%)       0m (0%)        1m (0%)      250Mi (1%)        750Mi (5%)      21Mi (0%)
ip-10-0-235-76.eu-west-2.compute.internal    kube-system   cert-exporter-daemonset-dzmgh                                        50m (3%)       150m (10%)     0m (0%)      50Mi (0%)         50Mi (0%)       11Mi (0%)
ip-10-0-235-76.eu-west-2.compute.internal    kube-system   net-exporter-h8pzp                                                   30m (2%)       0m (0%)        1m (0%)      60Mi (0%)         150Mi (1%)      13Mi (0%)
ip-10-0-235-76.eu-west-2.compute.internal    kube-system   capi-node-labeler-lmxf2                                              0m (0%)        0m (0%)        1m (0%)      0Mi (0%)          0Mi (0%)        11Mi (0%)
                                                                                                                                                                                                             
ip-10-0-70-202.eu-west-2.compute.internal    *             *                                                                    1305m (93%)    1650m (117%)   148m (10%)   1370Mi (9%)       2300Mi (16%)    3163Mi (22%)
ip-10-0-70-202.eu-west-2.compute.internal    kube-system   kube-apiserver-ip-10-0-70-202.eu-west-2.compute.internal             250m (17%)     0m (0%)        34m (2%)     0Mi (0%)          0Mi (0%)        954Mi (6%)
ip-10-0-70-202.eu-west-2.compute.internal    kube-system   kube-controller-manager-ip-10-0-70-202.eu-west-2.compute.internal    200m (14%)     0m (0%)        1m (0%)      0Mi (0%)          0Mi (0%)        30Mi (0%)
ip-10-0-70-202.eu-west-2.compute.internal    kube-system   aws-cloud-controller-manager-s6mgg                                   200m (14%)     400m (28%)     1m (0%)      300Mi (2%)        600Mi (4%)      19Mi (0%)
ip-10-0-70-202.eu-west-2.compute.internal    kube-system   k8s-dns-node-cache-7lf84                                             100m (7%)      0m (0%)        1m (0%)      100Mi (0%)        0Mi (0%)        23Mi (0%)
ip-10-0-70-202.eu-west-2.compute.internal    kube-system   etcd-ip-10-0-70-202.eu-west-2.compute.internal                       100m (7%)      0m (0%)        31m (2%)     100Mi (0%)        0Mi (0%)        312Mi (2%)
ip-10-0-70-202.eu-west-2.compute.internal    kube-system   cilium-vqwbq                                                         100m (7%)      0m (0%)        5m (0%)      10Mi (0%)         0Mi (0%)        202Mi (1%)
ip-10-0-70-202.eu-west-2.compute.internal    kube-system   etcd-k8s-res-count-exporter-94864df85-gclk2                          100m (7%)      100m (7%)      1m (0%)      250Mi (1%)        250Mi (1%)      25Mi (0%)
ip-10-0-70-202.eu-west-2.compute.internal    kube-system   kube-scheduler-ip-10-0-70-202.eu-west-2.compute.internal             100m (7%)      0m (0%)        3m (0%)      0Mi (0%)          0Mi (0%)        35Mi (0%)
ip-10-0-70-202.eu-west-2.compute.internal    kube-system   cert-exporter-daemonset-4x8bj                                        50m (3%)       150m (10%)     0m (0%)      50Mi (0%)         50Mi (0%)       11Mi (0%)
ip-10-0-70-202.eu-west-2.compute.internal    kube-system   node-exporter-node-exporter-269cw                                    50m (3%)       0m (0%)        1m (0%)      250Mi (1%)        750Mi (5%)      14Mi (0%)
ip-10-0-70-202.eu-west-2.compute.internal    kube-system   net-exporter-8flvr                                                   30m (2%)       0m (0%)        1m (0%)      60Mi (0%)         150Mi (1%)      13Mi (0%)
ip-10-0-70-202.eu-west-2.compute.internal    kube-system   promtail-fw27w                                                       25m (1%)       1000m (71%)    29m (2%)     250Mi (1%)        500Mi (3%)      58Mi (0%)
ip-10-0-70-202.eu-west-2.compute.internal    kube-system   capi-node-labeler-8nczt                                              0m (0%)        0m (0%)        1m (0%)      0Mi (0%)          0Mi (0%)        9Mi (0%)
                                                                                                                                                                                                             
ip-10-0-128-184.eu-west-2.compute.internal   *             *                                                                    1260m (90%)    1550m (110%)   262m (18%)   1370Mi (9%)       2050Mi (14%)    3146Mi (22%)
ip-10-0-128-184.eu-west-2.compute.internal   kube-system   kube-apiserver-ip-10-0-128-184.eu-west-2.compute.internal            250m (17%)     0m (0%)        58m (4%)     0Mi (0%)          0Mi (0%)        906Mi (6%)
ip-10-0-128-184.eu-west-2.compute.internal   kube-system   kube-controller-manager-ip-10-0-128-184.eu-west-2.compute.internal   200m (14%)     0m (0%)        1m (0%)      0Mi (0%)          0Mi (0%)        31Mi (0%)
ip-10-0-128-184.eu-west-2.compute.internal   kube-system   aws-cloud-controller-manager-x7qj5                                   200m (14%)     400m (28%)     2m (0%)      300Mi (2%)        600Mi (4%)      16Mi (0%)
ip-10-0-128-184.eu-west-2.compute.internal   kube-system   cilium-z9wn4                                                         100m (7%)      0m (0%)        7m (0%)      10Mi (0%)         0Mi (0%)        198Mi (1%)
ip-10-0-128-184.eu-west-2.compute.internal   kube-system   etcd-ip-10-0-128-184.eu-west-2.compute.internal                      100m (7%)      0m (0%)        40m (2%)     100Mi (0%)        0Mi (0%)        312Mi (2%)
ip-10-0-128-184.eu-west-2.compute.internal   kube-system   kube-scheduler-ip-10-0-128-184.eu-west-2.compute.internal            100m (7%)      0m (0%)        4m (0%)      0Mi (0%)          0Mi (0%)        36Mi (0%)
ip-10-0-128-184.eu-west-2.compute.internal   kube-system   k8s-dns-node-cache-7fpwn                                             100m (7%)      0m (0%)        1m (0%)      100Mi (0%)        0Mi (0%)        16Mi (0%)
ip-10-0-128-184.eu-west-2.compute.internal   kube-system   ebs-csi-controller-7d8f8688c7-j77fx                                  55m (3%)       0m (0%)        7m (0%)      250Mi (1%)        0Mi (0%)        65Mi (0%)
ip-10-0-128-184.eu-west-2.compute.internal   kube-system   node-exporter-node-exporter-t7cdc                                    50m (3%)       0m (0%)        1m (0%)      250Mi (1%)        750Mi (5%)      13Mi (0%)
ip-10-0-128-184.eu-west-2.compute.internal   kube-system   cert-exporter-daemonset-fpk8v                                        50m (3%)       150m (10%)     0m (0%)      50Mi (0%)         50Mi (0%)       12Mi (0%)
ip-10-0-128-184.eu-west-2.compute.internal   kube-system   net-exporter-8wdfq                                                   30m (2%)       0m (0%)        1m (0%)      60Mi (0%)         150Mi (1%)      12Mi (0%)
ip-10-0-128-184.eu-west-2.compute.internal   kube-system   promtail-5dd2m                                                       25m (1%)       1000m (71%)    25m (1%)     250Mi (1%)        500Mi (3%)      55Mi (0%)
ip-10-0-128-184.eu-west-2.compute.internal   kube-system   capi-node-labeler-s5cqn                                              0m (0%)        0m (0%)        1m (0%)      0Mi (0%)          0Mi (0%)        7Mi (0%)
```

**Comment by JosephSalisbury** [comment]
so, looks like cloud-controller-manager, dns-node-cache, and cluster-autoscaler all have silly requests, i'll follow up there next

**Comment by JosephSalisbury** [comment]
```
2024-05-01 14:21:16 
$ KUBECONFIG=~/Desktop/kubeconfig kubectl -n=giantswarm get pod chart-operator-66655fd845-fr8cl -o yaml | grep -A 5 resources 
    vpaUpdates: 'Pod resources updated by chart-operator: container 0: memory request,
      cpu request'
  creationTimestamp: "2024-05-01T13:06:52Z"
  generateName: chart-operator-66655fd845-
  labels:
    app: chart-operator
--
    resources:
      requests:
        cpu: 250m
        memory: 262144k
    securityContext:
      allowPrivilegeEscalation: false
--
    resources: {}
    securityContext:
      allowPrivilegeEscalation: false
      capabilities:
        drop:
        - ALL
```

and chart-operator there too

**Comment by JosephSalisbury** [comment]
also

```
$ KUBECONFIG=~/Desktop/kubeconfig kubectl -n=kube-system get pod coredns-controlplane-84bd7c7655-rl5hj -o yaml | grep -A 5 resources
    resources:
      limits:
        memory: 1Gi
      requests:
        cpu: 250m
        memory: 512Mi
```

coredns-controlplane (in addition to node cache)

**Comment by JosephSalisbury** [comment]
note: changed back to only updating control plane node instance types, reducing number of changes in this PR

**Comment by JosephSalisbury** [comment]
```
2024-05-09 14:58:43 
$ KUBECONFIG=~/Desktop/kubeconfig kubectl resource-capacity --node-labels=kubernetes.io/role=master -p --sort cpu.request -u            
NODE                                         NAMESPACE     POD                                                                  CPU REQUESTS   CPU LIMITS     CPU UTIL     MEMORY REQUESTS   MEMORY LIMITS   MEMORY UTIL
*                                            *             *                                                                    3470m (82%)    4750m (113%)   361m (8%)    4322Mi (10%)      12024Mi (28%)   7850Mi (18%)
                                                                                                                                                                                                             
ip-10-0-245-96.eu-west-2.compute.internal    *             *                                                                    1310m (93%)    1650m (117%)   129m (9%)    1870Mi (13%)      4100Mi (29%)    2748Mi (19%)
ip-10-0-245-96.eu-west-2.compute.internal    kube-system   kube-apiserver-ip-10-0-245-96.eu-west-2.compute.internal             250m (17%)     0m (0%)        29m (2%)     0Mi (0%)          0Mi (0%)        1005Mi (7%)
ip-10-0-245-96.eu-west-2.compute.internal    kube-system   kube-controller-manager-ip-10-0-245-96.eu-west-2.compute.internal    200m (14%)     0m (0%)        1m (0%)      0Mi (0%)          0Mi (0%)        23Mi (0%)
ip-10-0-245-96.eu-west-2.compute.internal    kube-system   cluster-autoscaler-6684975869-hjbfn                                  200m (14%)     0m (0%)        2m (0%)      400Mi (2%)        400Mi (2%)      62Mi (0%)
ip-10-0-245-96.eu-west-2.compute.internal    kube-system   kube-scheduler-ip-10-0-245-96.eu-west-2.compute.internal             100m (7%)      0m (0%)        1m (0%)      0Mi (0%)          0Mi (0%)        26Mi (0%)
ip-10-0-245-96.eu-west-2.compute.internal    kube-system   cilium-zpz6p                                                         100m (7%)      0m (0%)        5m (0%)      10Mi (0%)         0Mi (0%)        201Mi (1%)
ip-10-0-245-96.eu-west-2.compute.internal    kube-system   etcd-k8s-res-count-exporter-94864df85-74q5j                          100m (7%)      100m (7%)      1m (0%)      250Mi (1%)        250Mi (1%)      18Mi (0%)
ip-10-0-245-96.eu-west-2.compute.internal    kube-system   etcd-ip-10-0-245-96.eu-west-2.compute.internal                       100m (7%)      0m (0%)        24m (1%)     100Mi (0%)        0Mi (0%)        260Mi (1%)
ip-10-0-245-96.eu-west-2.compute.internal    kube-system   ebs-csi-controller-5679c47d77-wplnx                                  55m (3%)       0m (0%)        1m (0%)      250Mi (1%)        0Mi (0%)        62Mi (0%)
ip-10-0-245-96.eu-west-2.compute.internal    kube-system   node-exporter-node-exporter-swhmm                                    50m (3%)       0m (0%)        1m (0%)      250Mi (1%)        750Mi (5%)      11Mi (0%)
ip-10-0-245-96.eu-west-2.compute.internal    kube-system   aws-cloud-controller-manager-8tpnn                                   50m (3%)       400m (28%)     2m (0%)      250Mi (1%)        2000Mi (14%)    16Mi (0%)
ip-10-0-245-96.eu-west-2.compute.internal    kube-system   cert-exporter-daemonset-dbn29                                        50m (3%)       150m (10%)     0m (0%)      50Mi (0%)         50Mi (0%)       9Mi (0%)
ip-10-0-245-96.eu-west-2.compute.internal    kube-system   net-exporter-gz5s7                                                   30m (2%)       0m (0%)        1m (0%)      60Mi (0%)         150Mi (1%)      11Mi (0%)
ip-10-0-245-96.eu-west-2.compute.internal    kube-system   promtail-l5xxv                                                       25m (1%)       1000m (71%)    9m (0%)      250Mi (1%)        500Mi (3%)      47Mi (0%)
ip-10-0-245-96.eu-west-2.compute.internal    kube-system   capi-node-labeler-ts557                                              0m (0%)        0m (0%)        1m (0%)      0Mi (0%)          0Mi (0%)        7Mi (0%)
                                                                                                                                                                                                             
ip-10-0-171-116.eu-west-2.compute.internal   *             *                                                                    1205m (86%)    1550m (110%)   143m (10%)   1482Mi (10%)      4474Mi (32%)    3108Mi (22%)
ip-10-0-171-116.eu-west-2.compute.internal   kube-system   kube-apiserver-ip-10-0-171-116.eu-west-2.compute.internal            250m (17%)     0m (0%)        44m (3%)     0Mi (0%)          0Mi (0%)        1379Mi (9%)
ip-10-0-171-116.eu-west-2.compute.internal   kube-system   coredns-controlplane-84bd7c7655-jn89g                                250m (17%)     0m (0%)        1m (0%)      512Mi (3%)        1024Mi (7%)     20Mi (0%)
ip-10-0-171-116.eu-west-2.compute.internal   kube-system   kube-controller-manager-ip-10-0-171-116.eu-west-2.compute.internal   200m (14%)     0m (0%)        6m (0%)      0Mi (0%)          0Mi (0%)        100Mi (0%)
ip-10-0-171-116.eu-west-2.compute.internal   kube-system   cilium-78nq4                                                         100m (7%)      0m (0%)        5m (0%)      10Mi (0%)         0Mi (0%)        212Mi (1%)
ip-10-0-171-116.eu-west-2.compute.internal   kube-system   etcd-ip-10-0-171-116.eu-west-2.compute.internal                      100m (7%)      0m (0%)        30m (2%)     100Mi (0%)        0Mi (0%)        240Mi (1%)
ip-10-0-171-116.eu-west-2.compute.internal   kube-system   kube-scheduler-ip-10-0-171-116.eu-west-2.compute.internal            100m (7%)      0m (0%)        2m (0%)      0Mi (0%)          0Mi (0%)        32Mi (0%)
ip-10-0-171-116.eu-west-2.compute.internal   kube-system   node-exporter-node-exporter-8fsmc                                    50m (3%)       0m (0%)        1m (0%)      250Mi (1%)        750Mi (5%)      14Mi (0%)
ip-10-0-171-116.eu-west-2.compute.internal   kube-system   cert-exporter-daemonset-bc2tb                                        50m (3%)       150m (10%)     0m (0%)      50Mi (0%)         50Mi (0%)       8Mi (0%)
ip-10-0-171-116.eu-west-2.compute.internal   kube-system   aws-cloud-controller-manager-ghvk2                                   50m (3%)       400m (28%)     2m (0%)      250Mi (1%)        2000Mi (14%)    16Mi (0%)
ip-10-0-171-116.eu-west-2.compute.internal   kube-system   net-exporter-pnn7c                                                   30m (2%)       0m (0%)        1m (0%)      60Mi (0%)         150Mi (1%)      11Mi (0%)
ip-10-0-171-116.eu-west-2.compute.internal   kube-system   promtail-tf5cs                                                       25m (1%)       1000m (71%)    9m (0%)      250Mi (1%)        500Mi (3%)      52Mi (0%)
ip-10-0-171-116.eu-west-2.compute.internal   kube-system   capi-node-labeler-m8tt7                                              0m (0%)        0m (0%)        1m (0%)      0Mi (0%)          0Mi (0%)        8Mi (0%)
                                                                                                                                                                                                             
ip-10-0-77-188.eu-west-2.compute.internal    *             *                                                                    955m (68%)     1550m (110%)   91m (6%)     970Mi (6%)        3450Mi (24%)    1995Mi (14%)
ip-10-0-77-188.eu-west-2.compute.internal    kube-system   kube-apiserver-ip-10-0-77-188.eu-west-2.compute.internal             250m (17%)     0m (0%)        17m (1%)     0Mi (0%)          0Mi (0%)        557Mi (4%)
ip-10-0-77-188.eu-west-2.compute.internal    kube-system   kube-controller-manager-ip-10-0-77-188.eu-west-2.compute.internal    200m (14%)     0m (0%)        2m (0%)      0Mi (0%)          0Mi (0%)        25Mi (0%)
ip-10-0-77-188.eu-west-2.compute.internal    kube-system   etcd-ip-10-0-77-188.eu-west-2.compute.internal                       100m (7%)      0m (0%)        18m (1%)     100Mi (0%)        0Mi (0%)        216Mi (1%)
ip-10-0-77-188.eu-west-2.compute.internal    kube-system   cilium-vt7db                                                         100m (7%)      0m (0%)        4m (0%)      10Mi (0%)         0Mi (0%)        189Mi (1%)
ip-10-0-77-188.eu-west-2.compute.internal    kube-system   kube-scheduler-ip-10-0-77-188.eu-west-2.compute.internal             100m (7%)      0m (0%)        1m (0%)      0Mi (0%)          0Mi (0%)        25Mi (0%)
ip-10-0-77-188.eu-west-2.compute.internal    kube-system   aws-cloud-controller-manager-tm8qs                                   50m (3%)       400m (28%)     2m (0%)      250Mi (1%)        2000Mi (14%)    20Mi (0%)
ip-10-0-77-188.eu-west-2.compute.internal    kube-system   cert-exporter-daemonset-q4655                                        50m (3%)       150m (10%)     1m (0%)      50Mi (0%)         50Mi (0%)       11Mi (0%)
ip-10-0-77-188.eu-west-2.compute.internal    kube-system   node-exporter-node-exporter-rvc5r                                    50m (3%)       0m (0%)        4m (0%)      250Mi (1%)        750Mi (5%)      13Mi (0%)
ip-10-0-77-188.eu-west-2.compute.internal    kube-system   net-exporter-kdhvq                                                   30m (2%)       0m (0%)        1m (0%)      60Mi (0%)         150Mi (1%)      8Mi (0%)
ip-10-0-77-188.eu-west-2.compute.internal    kube-system   promtail-6hpv4                                                       25m (1%)       1000m (71%)    8m (0%)      250Mi (1%)        500Mi (3%)      52Mi (0%)
ip-10-0-77-188.eu-west-2.compute.internal    kube-system   capi-node-labeler-v54x7                                              0m (0%)        0m (0%)        1m (0%)      0Mi (0%)          0Mi (0%)        8Mi (0%)
```

**Comment by JosephSalisbury** [comment]
wild

```
2024-05-09 15:10:36 
$ KUBECONFIG=~/Desktop/kubeconfig kubectl -n=giantswarm get vpa chart-operator -o yaml
apiVersion: autoscaling.k8s.io/v1
kind: VerticalPodAutoscaler
metadata:
  annotations:
    meta.helm.sh/release-name: chart-operator
    meta.helm.sh/release-namespace: giantswarm
  creationTimestamp: "2024-05-09T13:51:59Z"
  generation: 1
  labels:
    app: chart-operator
    app.giantswarm.io/branch: HEAD
    app.giantswarm.io/commit: 4a1d150dd627946f742a68257eeab24155f57dee
    app.kubernetes.io/instance: chart-operator
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/name: chart-operator
    app.kubernetes.io/version: 3.2.1
    helm.sh/chart: chart-operator-3.2.1
  name: chart-operator
  namespace: giantswarm
  resourceVersion: "4366"
  uid: 06b5ad01-bc2e-486c-8c57-1583b9f139a8
spec:
  resourcePolicy:
    containerPolicies:
    - containerName: chart-operator
      controlledResources:
      - cpu
      - memory
      maxAllowed:
        cpu: 1000m
        memory: 1000Mi
      minAllowed:
        cpu: 250m
        memory: 250Mi
      mode: Auto
  targetRef:
    apiVersion: apps/v1
    kind: Deployment
    name: chart-operator
  updatePolicy:
    updateMode: Auto
status:
  conditions:
  - lastTransitionTime: "2024-05-09T13:54:21Z"
    status: "True"
    type: RecommendationProvided
  recommendation:
    containerRecommendations:
    - containerName: chart-operator
      lowerBound:
        cpu: 250m
        memory: 262144k
      target:
        cpu: 671m
        memory: 262144k
      uncappedTarget:
        cpu: 671m
        memory: 262144k
      upperBound:
        cpu: "1"
        memory: 1000Mi
```

**Comment by JosephSalisbury** [comment]
i'm gonna revisit this PR once we get the requests for the apps sorted, it's just hanging rn

**Pull request closed without merging**